worker/credentialvalidator/util_test.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2018 Canonical Ltd.
```

Comment:
I've really gone off the name util. What does this really contain? 

mocks_test.go ??